spec/strava/api/client/endpoints/uploads/create_upload_spec.rb

```diff
@@ -17,4 +17,18 @@
     expect(upload.status).to eq 'Your activity is still being processed.'
     expect(upload.activity_id).to eq nil
   end
+  it 'returns a Strava::Errors::UploadFailed on upload error', vcr: :all do
```

Comment:
What’s this vcr: all? Are you missing a VCR file for the error explicitly recorded in the commit? Make sure that doesn’t have a key, etc.